src/block/Dirt.php

```diff
@@ -59,7 +59,12 @@ public function setDirtType(DirtType $dirtType) : self{
 
 	public function onInteract(Item $item, int $face, Vector3 $clickVector, ?Player $player = null, array &$returnedItems = []) : bool{
 		$world = $this->position->getWorld();
-		if($face === Facing::UP && $item instanceof Hoe){
+		if($face !== Facing::DOWN && $item instanceof Hoe){
+			$down = $this->getSide(Facing::UP);
```

Comment:
Its up, not down, no ?

Reply:
Oh, right. I didn't notice the variable name from the copy paste